### DISCUSSION TITLE
Parental leave: this repository will be under maintenance until the principal maintainer is back (Aug 1st, 23)

### DISCUSSION THREAD

**Discussion by mdelapenya** [testcontainers/testcontainers-go#653] [Announcements]
Hi Testcontainers for Go community!

## First period of parental leave (Dec 22 - Jan 23)

Here I'm writing down this issue, counting the tick-tacks of the clock, waiting for our second son 🐥 🐣 , who can eventually come in 2 weeks max.

I'd like to share that, because of that, I'm transitioning the maintenance of the repository to my colleagues at AtomicJar, @kiview @eddumelendez and @HofmeisterAn, who are the principal maintainers for the Java and .NET flavours of this awesome library.

They are great engineers that will do their best during my leave: responding issues, reviewing PRs, or answering on Slack threads. So please do not be too pushy with them, as they will still keep an eye on the Go library apart from their own duties with the other language projects.

My leave will take around 6 weeks starting the next day of the due date, so I will be probably back at the beginning of February.

In the mean time, I'll update this discussion when the baby comes, probably with a cute picture 👶 , meaning the starting of my leave.

## Second period of parental leave (Jul 23)

I'm taking the second part of my parental leave starting June 26th, so this repo will be under maintenance until Aug 1st, which is the date I'm back from leave.

**Comment by hezhangjian** [comment]
congratulations :)

**Comment by kiview** [comment]
Just as an update for everyone in the community, @mdelapenya is on leave since this week (12th of December) 🥳

**Comment by mdelapenya** [comment]
Another update, I'm back!

I'll use this week to understand what went on during this 6 weeks, and hopefully will be back to full-speed at the end of it.

Thank you very much for your support during this time.

![image](https://media.giphy.com/media/42YlR8u9gV5Cw/giphy.gif)

**Comment by mdelapenya** [comment]
My parental leave period has finished. I've enjoyed a lot the time with my two kids, but now it's time to rock with Testcontainers for Go! 🚀 

Therefore I'm closing this discussion thread, un-pinning it from the repo
